examples/gridplot/multigraphic_gridplot.py

```diff
@@ -54,12 +55,7 @@ def make_circle(center, radius: float, n_points: int = 75) -> np.ndarray:
     circles.append(make_circle(center, 5, n_points=75))
 
 # things like class labels, cluster labels, etc.
-cmap_transform = [
-    0, 1, 1, 2,
-    0, 0, 1, 1,
-    2, 2, 8, 3,
-    1, 9, 1, 5
-]
+cmap_transform = [0, 1, 1, 2, 0, 0, 1, 1, 2, 2, 8, 3, 1, 9, 1, 5]
```

Comment:
We don't black the examples because it's useful to manually set things like this list sometimes so the user has a better visual understanding.